PSReadLine/ReadLine.vi.cs

```diff
@@ -849,6 +849,28 @@ public static void DeletePreviousLines(ConsoleKeyInfo? key = null, object arg =
             }
         }
 
+        /// <summary>
+        /// Delete from the current logical line to the n-th requested logical line in a multiline buffer
+        /// </summary>
+        private static void DeleteRelativeLines(ConsoleKeyInfo? key = null, object arg = null)
+        {
+            if (TryGetArgAsInt(arg, out var requestedLineNumber, 1))
+            {
+                var currentLineIndex = _singleton.GetLogicalLineNumber() - 1;
+                var requestedLineIndex = Math.Max(requestedLineNumber, _singleton.GetStatusLineCount()) - 1;

```

Comment:
Why comparing `requestedLineNumber` with `_singleton.GetStatusLineCount()`? `_singleton.GetStatusLineCount()` gets you the physical line count of the `_statusLinePrompt` + `_statusBuffer`. Did you intend to use `GetLogicalLineCount`?

Reply:
This is indeed an error.